### ISSUE TITLE
Error when testing on commit: Cannot read property 'comments_url' of undefined

### ISSUE THREAD

**Issue by marlon360** [justinribeiro/lighthouse-action#8]
If I run this action on a commit (not a PR) I got following error:
`Cannot read property 'comments_url' of undefined`

**Comment by justinribeiro** [comment]
That is a good catch; comment_url isn't set for a given commit. I'll put a fix in for that case.

**Comment by marlon360** [comment]
Thanks!
Would be nice to have the option to comment on every new commit, not just a PR.

**Comment by pascalandy** [comment]
Same error here:

## yml

```
      - 
        name: Saving Lighthouse Audit Artifacts
        uses: actions/upload-artifact@master
        with:
          name: lighthouse-artifacts
          path: './results'
```

## logs

```
Warning: secret not defined; PR comment reporting disabled.
CDP: network conditions set to WPT threegfast profile.
CDP: network conditions set to WPT threegfast profile.
CDP: network conditions set to WPT threegfast profile.
CDP: network conditions set to WPT threegfast profile.
CDP: network conditions set to WPT threegfast profile.

[Lighthouse](https://developers.google.com/web/tools/lighthouse/) report for the changes in this PR:

WPT Profile: threegfast

| Category | Score |
| ------------- | ------------- |
| Performance | 100 | 
| Accessibility | 87 | 
| Best Practices | 100 | 
| SEO | 90 | 
| Progressive Web App | 56.00000000000001 | 

| Measure | Timing |
| ------------- | ------------- |
| First Contentful Paint | 1.3 s | 
| First Meaningful Paint | 1.3 s | 
| Max Potential First Input Delay | 260 ms | 
| Time to Interactive | 2.0 s | 
| First CPU Idle | 2.0 s | 
| Speed Index | 1.7 s | 

_Tested with Lighthouse v5.6.0 via [lighthouse-action](https://github.com/justinribeiro/lighthouse-action)_
Error: Cannot read property 'comments_url' of undefined
```

**Comment by geekkun** [comment]
Same here :(

**Comment by justinribeiro** [comment]
I'll circle back this week on this; apologizes for delay!

**Comment by justinribeiro** [comment]
Patched in v2.0.0 release.
